testcases/open_posix_testsuite/conformance/interfaces/munmap/2-1.c

```diff
@@ -4,12 +4,15 @@
  * of this license, see the COPYING file at the top level of this
  * source tree.
  *
- * If there are no
- * mappings in the specified address range, then munmap() has no effect.
+ * If there are no mappings in the specified address range, then munmap() has
+ * no effect. To get a valid address range which is safe to call munmap() on
+ * we first map some arbitrary memory allowing the OS to select the address
+ * then unmap it. We then call munmap() on the same address again to perform
+ * the test.
  *
  */
 
-#define _XOPEN_SOURCE 600
+#define _DEFAULT_SOURCE
 
```

Comment:
What do we need to define the `_DEFAULT_SOURCE` for?

The `_XOPEN_SOURCE` is a macro that is used to export some extended POSIX functionality on older systems. The `_DEFAULT_SOURCE` is glibc specific and hence should not be used in Open Posix Testsuite. Moreover as far as I can tell not defining anything should have the same effect as defining `_DEFAULT_SOURCE`.

Otherwise the rest looks OK.



Otherwise